lib/versioning/VersionID.js

```diff
@@ -202,8 +215,19 @@ function base62Decode(str) {
         const enc2 = str.substring(start, start + B62V_EPAD.length);
         const orig2 = base62Integer.decode(enc2);
         start += B62V_EPAD.length;
-        const enc3 = str.substring(start);
+        let enc3 = str.substring(start);
+        // take off null bytes padding only from the end
```

Comment:
You can use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/lastIndexOf to find the last character you need. TrimEnd might work but doesn't seem like it unless \u0000 counts as white space.

Reply:
we're not trying to find the last index with null but the first index with null. TrimEnd doesn't work unfortunately.

Reply:
oh well the first index would just be `indexOf` unless you specifically needed to start from the end for some reason.

Reply:
there's a comment above with the benchmarks.

Reply:
I originally did it with just regex but Vianney suggested to optimize

Reply:
oh yea missed it, sorry